### DISCUSSION TITLE
ETA for new release?

### DISCUSSION THREAD

**Discussion by rustprooflabs** [ULB-CoDE-WIT/MobilityDB#355] [Q&A] [answered]
Is there an ETA for a new release of MobilityDB? While trying to get started I found multiple blocking issues with v1.0 / master branch (see https://github.com/MobilityDB/MobilityDB/pull/351 and https://github.com/MobilityDB/MobilityDB/issues/352) before realizing they were all fixed in `develop` and have been for quite a while. The issues I ran into were related to using Ubuntu 22.04 / Postgres 15 and currently block consideration for production use.

Thanks!

**Comment by estebanzimanyi** [comment]
Many thanks for your interest in MobilityDB. Our current plan is to wait that the packaging for Debian
https://www.postgresql.org/message-id/CAOMoQbSGU0PJmOd4n76B8_DG4LTj9FHqEAxcdMCoA1gNXabCvg%40mail.gmail.com
and YUM
https://www.postgresql.org/message-id/CAPqRbE7GDMhQB1jEmuncCJZnGiz5RbEpubsdJRT6kcA5eRfaYg%40mail.gmail.com
is finished to launch the release. Probably due to summer holidays, things are evolving slowly in this respect.

I have therefore two questions:
* Is this plan acceptable for you to consider to put in production ? Any suggestions in this regard are most welcome.
* Do you have any suggestion how to speed up the packaging process ? 

Regards

**Reply by rustprooflabs** [reply]
Hi @estebanzimanyi, thank you for your response!  Having packages to install should be a big benefit to encourage wider adoption of the project. I have little experience in packaging so can't offer much in that realm.

That said, I'm okay installing from source as long as I have a known, stable branch/tag to use.  Would the project be willing to provide more, smaller releases (merges to `master`) to support fixes that address supporting new OS/Postgres versions as time moves on?  While a tagged version would be nice, merges into the `master` branch would be enough.  I just can't rely on a `develop` branch.

My concern is the cadence of getting these fixes/updates into a stable place.  More than a year seems like a long time for a `.1` release, or a merge to the `master` branch.  The change to enable using Postgres 15 (https://github.com/MobilityDB/MobilityDB/issues/218) has been in `develop` for 14 months, I assume the fix for Ubuntu 22.04 support was added in a similar timeframe.  I see Postgres 16 support is also waiting in #333! :smile:  I typically start updating at least some servers to the latest Postgres version within the first days after it being released.

In the shorter term, would you consider a 1.1.rc1 release?   If the main thing that's waiting is packaging, I'd love to start testing that.

**Reply by estebanzimanyi** [reply, marked as answer]
We have just released
https://github.com/MobilityDB/MobilityDB/releases/tag/v1.1.0-alpha
Maybe you can already use this version for testing. 
Please be aware that the binary format has considerably changed from 1.0 to 1.1 so it is necessary to reload the databases containing columns using MobilityDB types.
But on the plus side, you have a much more powerful and optimized backend with many enhanced possibilities for doing mobility analytics.
Please let us know how we can help you in migrating from 1.0 to 1.1

**Comment by estebanzimanyi** [comment]
MobilityDB has had several releases since this question, and it is packaged for PostgreSQL on both the Debian (apt.postgresql.org) and YUM (yum.postgresql.org) repositories, so the version and packaging concerns raised here are resolved. Closing as outdated; please open a new issue for any current build problem.
